Source/Scene/Implicit3DTileContent.js

```diff
@@ -374,13 +386,26 @@ function deriveChildTile(
     boundingVolume: boundingVolume,
     geometricError: childGeometricError,
     refine: implicitTileset.refine,
-    content: contentJson,
   };
 
+  if (contentJsons.length === 1) {
+    tileJson.content = contentJsons[0];
+  } else if (contentJsons.length > 1) {
+    var multipleContentExtension = {
+      "3DTILES_multiple_contents": {
+        content: contentJsons,
+      },
+    };
+    var existingExtensions = implicitTileset.tileHeader.extensions;
+    tileJson.extensions = combine(multipleContentExtension, existingExtensions);
+  }
+
   var childTile = makeTile(
     implicitContent,
     implicitTileset.baseResource,
-    tileJson,
+    // combine() is used to pass through any additional properties the
+    // user specified such as extras or extensions
+    combine(tileJson, implicitTileset.tileHeader),
```

Comment:
Would this potentially overwrite the extensions set in the line above?

`tileJson.extensions = combine(multipleContentExtension, existingExtensions);`

Reply:
ah actually it's _because_

But I realized that it's more readable to just use:

```json
var deep = true;
combine(tileJson, implicitTileset.tileHeader, deep);
```

Note that `combine(A, B)` updates from _right-to-left_, with properties from A clobbering those from B if there are collisions.